### DISCUSSION TITLE
Imported Skins stutter too much

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#22466] [Q&A]
Whenever I use my imported skins the gameplay stutter too much and sometimes the gameplay freeze then suddenly resume, and was like fast forward when it resumed. But when I use lazer's Default skin, it's not stuttering too much.

I'm not sure yet if this problem was from my phone or from the game

**Comment by peppy** [comment]
Provide your skin.

**Reply by Shirou27** [reply]
https://drive.google.com/file/d/1KAhrAqp8jZi75_SiilspiTswoaZs6LnV/view and https://drive.google.com/file/d/1VGxrIWcFRgc4nhHRv4KB22MsUfTMBTRb/view

**Reply by mk56-spn** [reply]
Opened it up to analyse and from what was mentioned on another thread regarding poor skin performance i think the files named "scorebar-colour-(number)"  are going to be the culprits here, since it has 300 unique frames, since i assume having to run that many frames is taxing for low end devices, skin also seems to be on the heftier side overall, but not excessively so.

**Reply by peppy** [reply]
The skin just has an insane number of animation frames, not just on the score bar but also judgements and skip button and more.

We'll probably get around to optimising this kind of case at some point, but it's also going to be hard to resolve for mobile devices with limited GPU memory.

**Reply by Shirou27** [reply]
I'm gonna look forward to it. Thanks
